live/index.html

```diff
@@ -0,0 +1,747 @@
+	<meta name="image" content="http://hackhpi.org/site/templates/images/facebook-preview.png">
+	<link rel="canonical" href="https://live.hackhpi.org/">
+	<meta name="generator" content="ProcessWire 3.0.15">
+	<meta name="author" content="HackHPI">
+	<meta property="og:site_name" content="HackHPI">
+	<meta property="og:title" content="HackHPI 2018 - about://environment">
+	<meta property="og:url" content="https://live.hackhpi.org/">
+	<meta property="og:description" content="Be a green hero! Come join our Hackathon about://environment on 26th and 27th of May 2018 at Hasso Plattner Institute and work towards a more sustainable world.">
+	<meta property="og:type" content="website">
+	<meta property="og:image" content="http://hackhpi.org/site/templates/images/facebook-preview.png">
+	<meta name="twitter:card" content="summary">
+	<meta name="twitter:site" content="@HackHPI">
+	<meta name="twitter:title" content="HackHPI 2018 - about://environment">
+	<meta name="twitter:url" content="https://live.hackhpi.org/">
+	<meta name="twitter:description" content="Be a green hero! Come join our Hackathon about://environment on 26th and 27th of May 2018 at Hasso Plattner Institute and work towards a more sustainable world.">
+	<meta name="twitter:image" content="http://hackhpi.org/site/templates/images/facebook-preview.png">
+
+</head>
+
+<body>
+	<section class="container-fluid bg-live" id="header">
+		<div class="container">
+			<div class="row">
+				<div class="col-md-12">
+					<h1>
+						<span class="live">HackHPI 2018 - about://environment</span>
+					</h1>
+					<h2 id="time">Thursday, 18th of January 2018 09:23:48 am</h2>
+				</div>
+			</div>
+		</div>
+	</section>
+
+	<section class="container-fluid bg-live">
+		<div class="container">
+
+			<div class="row">
+				<div class="col-md-12">
+					<h2>Important Information</h2>
+				</div>
+			</div>
+
+
+			<div class="row">
+				<div class="col-md-5">
+					<div class="faq-question">
+						<h3 class="text-orange">Do you have WIFI?</h3>
+						<p>Yes. Here are the credentials:</p>
+
+						<p>
+							<strong>SSID:</strong> HackHPI
+							<br>
+							<strong>Password:</strong> TBA</p>
+
+						<p>If your university uses EduRoam you can use that as well.</p>
+					</div>
+					<div class="faq-question">
+						<h3 class="text-orange">What if I don't have a team?</h3>
+						<p>Don't worry! At the end of our opening we will have a "match making" where you can find each other based on the idea
+							you want to work on. We will explain it in detail on Saturday.</p>
+
+						<p>Or you can use the  <a href="https://hackhpi2018.slack.com/messages/matchmaking" target="_blank">#matchmaking</a> channel in our Slack channel already to pitch your idea and find team mates&nbsp;:)</p>
+
+						<p>Teams should have a size of not more than 3-4.</p>
```

Comment:
According to current discussion in #internal, please make this more firm, e.g. "maximum team size is 4, we encourage teams of 3-4" and maybe create an own question for it. fyi @FWirtz